src/sortStudents.ts

```diff
@@ -1,16 +1,96 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+type Callback = (grades: number[]) => number;
+type Comparison = (a: string, b: string) => number;
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+const calc: Callback = (grades) => {
+  return grades.reduce((a: number, b: number) => a + b) / grades.length;
```

Comment:
Ты уже типизировал grades как массив чисел, когда описывал Callback
```suggestion
  return grades.reduce((a, b) => a + b) / grades.length;
```